puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -119,7 +119,8 @@
       # $_volume_clone_opts = "interleave=true"
       # $_cinder_volume_resource_name = "cinder-volume-clone"
     } else {
-      $_volume_clone_opts = undef
+      #$_volume_clone_opts = undef
```

Comment:
Better to simply remove lines that are no longer needed versus commenting them out.  This applies to a number of places in this patch